### PR TITLE
Add force_unlock/powermeter/telemetry to evsemanager

### PR DESCRIPTION
**Pull request by corneliusclaussen**
The force_unlock command should be used by OCPP if requested by central
management system.
The powermeter interface is proxied to the actual power meter in use by
the evse to charge the car. Connect the OCPP powermeter requirement to
this one instead of directly connecting it to the actual hw driver.

Signed-off-by: Cornelius Claussen <cc@pionix.de>

### CODE REVIEW

File: modules/EvseManager/EvseManager.cpp [hunk 1]
```diff
@@ -19,11 +21,16 @@ void EvseManager::ready() {
     });
 
     r_powermeter->subscribe_powermeter([this](json p) {
+        // Inform charger about current charging current. This is used for slow OC detection.
```
Comment (hikinggrass) [inline modules/EvseManager/EvseManager.cpp:24]: Is current changging current a typo?

File: modules/EvseManager/evse_energy_control/evse_manager_energy_controlImpl.hpp [hunk 2]
```diff
@@ -1,5 +1,3 @@
-// SPDX-License-Identifier: Apache-2.0
```
Comment (hikinggrass) [inline modules/EvseManager/evse_energy_control/evse_manager_energy_controlImpl.hpp:1]: How did the license header disappear here? Could this be a bug in ev-cli?

File: modules/EvseManager/powermeter/powermeterImpl.hpp [hunk 3]
```diff
@@ -0,0 +1,63 @@
+#ifndef POWERMETER_POWERMETER_IMPL_HPP
```
Comment (hikinggrass) [inline modules/EvseManager/powermeter/powermeterImpl.hpp:1]: Again, no license header

File: modules/YetiDriver/ld-ev.hpp [hunk 4]
```diff
@@ -1,5 +1,3 @@
-// SPDX-License-Identifier: Apache-2.0
```
Comment (hikinggrass) [inline modules/YetiDriver/ld-ev.hpp:1]: Missing license header.

**Review by hikinggrass — CHANGES_REQUESTED** [review]
Looks good, just a few missing license headers and maybe a typo.

**Review by a-w50 — COMMENTED** [review]
It seems, that part of the changes have been done with the old auto-generator and other parts with the new one.  Probably you want to do an force update with ev-cli.
Concerning the telemetry data, we can see here, that would we really should go on with global type definitions

File: modules/EvseManager/EvseManager.cpp (hunk 1 above)
Comment (corneliusclaussen) [reply modules/EvseManager/EvseManager.cpp:24]: No, english words just have too many meanings.

File: modules/EvseManager/powermeter/powermeterImpl.hpp (hunk 3 above)
Comment (corneliusclaussen) [reply modules/EvseManager/powermeter/powermeterImpl.hpp:3]: YetiDriver and EvseManager updated with edm mod update --force to fix license tags

File: modules/YetiDriver/ld-ev.hpp (hunk 4 above)
Comment (corneliusclaussen) [reply modules/YetiDriver/ld-ev.hpp:1]: solved by running edm mod update --force

File: modules/EvseManager/evse_energy_control/evse_manager_energy_controlImpl.hpp (hunk 2 above)
Comment (corneliusclaussen) [reply modules/EvseManager/evse_energy_control/evse_manager_energy_controlImpl.hpp:1]: solved by running edm mod update --force

**Comment by corneliusclaussen** [comment]
Currently this exposes a powermeter interface on evsemanager. This will need to be changed once we have json types, the powermeter var should be on the main interface instead of a seperate one. Seperating here creates a dependency mess: OCPP and friends should NEVER use a powermeter on its own, this would mean that e.g. start/end of session kwh values come from a different powermeter then the live values. Also with a 1:n dependency to evsemanagers it is even harder to match the correct power meters to the correct evsemanagers.

**Review by a-w50 — APPROVED** [review]
Licenses fixed now, perfect

**Review by hikinggrass — APPROVED** [review]

**Pull request merged**